x-pack/plugins/alerting/common/routes/rule/response/constants/latest.ts

```diff
@@ -0,0 +1,8 @@
+/*
```

Comment:
The rule response is like a shareable data schema for rules, since most of our rule endpoints will end up returning one or many rules, one would assume if the rule response schema changes for one, it would change for all of them. Hence why it's been extracted and re-used.